init-data.h

```diff
@@ -3,7 +3,10 @@
 
 
 #define DATA_SIZE 9u
-#define DATA_SIZE_H DATA_SIZE/2
+#define MEDIAN_WINDOW 3u
+#define MEDIAN_MID MEDIAN_WINDOW/2
+#define EXP_SMOOTH_WINDOW 1u
+#define NEW_MEAN_WINDOW 1u
 
```

Comment:
why do we need all of these windows?

Reply:
This windows are defined for each filter.
Cause we need only one raw data sample - window sizes are equal to 1.(for expSmoth and newMean filters).
I guess that this variables can be removed if I implement first comment.